packages/flutter_adaptive_scaffold/test/simulated_layout.dart

```diff
@@ -108,15 +108,32 @@ enum SimulatedLayout {
   medium(width: 800, navSlotKey: 'primaryNavigation'),
   mediumLarge(width: 1000, navSlotKey: 'primaryNavigation1'),
   large(width: 1200, navSlotKey: 'primaryNavigation2'),
-  extraLarge(width: 1600, navSlotKey: 'primaryNavigation3');
+  extraLarge(width: 1600, navSlotKey: 'primaryNavigation3'),
+  smallLandscape(width: 500, height: 400, navSlotKey: 'bottomNavigation'),
+  mediumLandscape(width: 800, height: 600, navSlotKey: 'primaryNavigation'),
+  mediumLargeLandscape(
+      width: 1100, height: 900, navSlotKey: 'primaryNavigation1'),
+  largeLandscape(width: 1400, height: 1000, navSlotKey: 'primaryNavigation2'),
+  extraLargeLandscape(
+      width: 1700, height: 1000, navSlotKey: 'primaryNavigation3'),
+  smallPortraitMediumLandscape(
```

Comment:
Should these `navSlotKey`s be unique or does it not affect anything?

Reply:
No they represent the active navigation key. So this is correct now.